sdk/storage/storage-blob/src/pollers/BlobStartCopyFromUrlPoller.ts

```diff
@@ -167,10 +183,11 @@ const update: BlobBeginCopyFromURLPollOperation["update"] = async function updat
 ): Promise<BlobBeginCopyFromURLPollOperation> {
   const state = this.state;
   const { blobClient, copySource, startCopyFromURLOptions } = state;
+  let result: BlobStartCopyFromURLResponse | BlobGetPropertiesResponse | undefined = undefined;
```

Comment:
don't need to initialize the variable to undefined, that happens if you just declare it.